apps/heureka/src/components/Services/ServicePanel/ServicePanel.tsx

```diff
@@ -0,0 +1,181 @@
+    imageName: version.ccrn,
+    imageVersion: version.version,
+    issueCounts: version.issueCounts,
+    serviceName: service?.name,
+  }))
+
+  return (
+    <Panel
+      heading={
+        <Stack gap="2">
+          <span>{capitalizeFirstLetter(service.name)} Service Overview</span>
+        </Stack>
+      }
+      opened={true}
+      onClose={onClose}
+      size="large"
+    >
+      <PanelBody>
+        <Stack gap="2" distribution="between" alignment="center" className="mb-2">
+          <ContentHeading
+            heading={`${capitalizeFirstLetter(service.name)} Image Versions${showTotalCount ? ` (${totalCount})` : ""}`}
+          />
+          <Button
+            disabled
+            variant="primary"
+            size="small"
+            onClick={() => {
+              // TODO: Navigate to service details page
+              onClose()
+            }}
+          >
+            Full Details
+          </Button>
+        </Stack>
+        <DataGrid columns={7}>
+          <DataGridRow>
+            <DataGridHeadCell>Image Name</DataGridHeadCell>
+            <DataGridHeadCell>Version</DataGridHeadCell>
+            <DataGridHeadCell>Critical</DataGridHeadCell>
+            <DataGridHeadCell>High</DataGridHeadCell>
+            <DataGridHeadCell>Medium</DataGridHeadCell>
+            <DataGridHeadCell>Low</DataGridHeadCell>
+            <DataGridHeadCell>Actions</DataGridHeadCell>
+          </DataGridRow>
+          {loading ? (
+            <EmptyDataGridRow colSpan={7}>Loading...</EmptyDataGridRow>
+          ) : formattedImageVersions?.length === 0 ? (
+            <EmptyDataGridRow colSpan={7}>No image versions available.</EmptyDataGridRow>
+          ) : error ? (
+            <EmptyDataGridRow colSpan={7}>
+              <Message variant="error">{error}</Message>
+            </EmptyDataGridRow>
+          ) : (
+            formattedImageVersions.map((version, index) => (
+              <DataGridRow key={index}>
+                <DataGridCell className="break-all overflow-hidden">
+                  <Stack gap="1" direction="vertical">
+                    <span>{version.imageName}</span>
+                    <Stack gap="1" alignment="center">
+                      <a
+                        href={`https://${version.imageName}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="hover:underline text-sm"
+                      >
+                        <Stack gap="1.5" alignment="center">
+                          <Icon icon="openInNew" size="16" color="jn-global-text" onClick={() => {}} />
+                          <span>Image repository</span>
+                        </Stack>
+                      </a>
+                    </Stack>
+                  </Stack>
+                </DataGridCell>
+                <DataGridCell>
+                  <Tooltip triggerEvent="hover" placement="top">
```

Comment:
Having looked at it some more I find that the truncated sha-hash of the image version doesn't look nice. My suggestion is to print it in full in subdued text color and smaller text size underneath the image name (where you currently already show the link to the image registry). if you have a version tag for the image you can display it here in this column